src/App/App.tsx

```diff
@@ -29,6 +30,9 @@ function App() {
             window.removeEventListener('keydown', handleCloseWindow);
         };
     }, [showNavbar]);
+    useEffect(() => {
+        login(12345678, 'asd123123').then((response) => console.log(response));
+    }, []);
```

Comment:
This is only for testing purposes, the login function will be used later

Reply:
Remove this block of code

Reply:
done